### PR TITLE
Completed requested changes and services.

### PR DESCRIPTION
**Pull request by ocampd1**

### CODE REVIEW

File: chucks-pick3-web/angular-app/src/app/app.module.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ import {SignUpConfirmationComponent} from 'src/app/components/sign-up-confirmati
 import {MemberService} from 'src/app/services/member.service/member.service';
 import { UserNameValidator } from './directives/user-name-validator.directive';
 import { PasswordValidator } from './directives/password.directive';
+import { GenerateCustomerService } from './services/generate-customer.service/generate-customer.service';
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/app.module.ts]: Rename GenerateCustomerService to MockCustomerService.

File: chucks-pick3-web/angular-app/src/app/components/customer-summary.component/customer-summary.component.ts [hunk 2]
```diff
@@ -12,7 +11,7 @@ import index from "@angular/cli/lib/cli";
 export class CustomerSummaryComponent implements OnInit {
   public customers: CustomerInfo [] = [];
 
-  constructor() {
+  constructor(public generateCustomerService: GenerateCustomerService) {
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/customer-summary.component/customer-summary.component.ts]: Rename `generateCustomerService` property to be just `customerService`.

File: chucks-pick3-web/angular-app/src/app/app.module.ts (hunk 1 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/app.module.ts]: Renamed to MockCustomerService.

File: chucks-pick3-web/angular-app/src/app/components/customer-summary.component/customer-summary.component.ts (hunk 2 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/customer-summary.component/customer-summary.component.ts]: Replaced generateCustomerService with customerService.

**Pull request merged**